workers/unity/ProjectSettings/BurstAotSettings_StandaloneWindows.json

```diff
@@ -1,11 +1,13 @@
 {
```

Comment:
do we need to update burst settings for other platforms?

Reply:
Yes! I thought enabling them for "standalone" would have done this... seems not

Reply:
Done